Dockerfile

```diff
@@ -49,10 +49,12 @@ ENV NGINX_VERSION=$NGINX_VERSION
 ADD nginx-$NGINX_VERSION.tar.gz /tmp/
 
 WORKDIR /tmp
-RUN ./configure && make
+RUN wget https://sourceforge.net/projects/pcre/files/pcre/8.44/pcre-8.44.tar.gz && tar xzvf pcre-8.44.tar.gz
```

Comment:
do we need to be verbose in the unpacking of this TARBALL? It's for automation right, so maybe that's just noise?